source/features/clean-repo-sidebar.tsx

```diff
@@ -36,6 +36,15 @@ async function hideEmptyPackages(): Promise<void> {
 	}
 }
 
+async function hideEmptyDeployments(): Promise<void> {
+	await domLoaded;
+
+	const deployments = await elementReady('.Layout-sidebar .BorderGrid-cell a[href*="/deployments"]', {waitForChildren: false})!;
+	if (deployments && deployments.parentElement!.parentElement!.querySelector('ul')!.children.length === 0) {
```

Comment:
This is unsafe, at this point it's easier to just use another standalone $() call